.github/workflows/js-tests.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: js-tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: deployment
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+          cache: npm
+      - run: yarn install
+      - run: yarn test
```

Comment:
Can we run the tests as a new step here? It'll be easier to debug in the later stage if there's any issue in the installation or the tests.